index.html

```diff
@@ -37,7 +37,158 @@ <h1 >I’m Esmatullah Glad to see you!</h1>
                     <li><a href=""><img src="icons/Vector2.svg" alt="social media icon"></a></li>
                 </ul>
             </div>
-        </div>    
+        </div>
+
+        <section id="works" class="container">
+            <div class="projects">
+                <div class="project-img">
+                    <img class="tonic-project-img" src="images/first project.svg" alt="">
+                </div>
+                <div class="project-info">
+                    <div class="project-name">
+                        <h2>Tonic</h2>
+                    </div>
+                    <div class="project-subtitle">        
+                        <p>CANOPY</p>
+                        <span></span>
+                        <p>Back End Dev</p>
+                        <span></span>
+                        <p>2015</p>
+                    </div>
+                    <div class="project-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <button class="porject-tect-btn">html</button>
+                        <button class="porject-tect-btn">css</button>
+                        <button class="porject-tect-btn">javaScript</button>
```

Comment:
- [ ] Great work aligning the project technologies:clap: However it will be better if we use `ul` tag for the technologies container and  `li` tag for each technology, so we can follow HTML & CSS semantic.